### PR TITLE
Docker healthchecks

### LINKED ISSUE #3569: add healthcheck to dockerfiles
Add a `HEALTHCHECK` command to all the dockerfiles for auto-restart functionality

### PR DESCRIPTION
**Pull request by normanrz**
- Adds healthchecks to all dockerfiles for auto-healing and starting order.
- Uses the new integrated fossildb healthcheck https://github.com/scalableminds/fossildb/pull/30

### Steps to test:
- `DOCKER_TAG=docker-health__7392 docker-compose pull webknossos webknossos-datastore webknossos-tracingstore`
- `DOCKER_TAG=docker-health__7392 docker-compose up webknossos webknossos-datastore webknossos-tracingstore`
- Check `docker ps` whether all services are reported healthy (after ~1min each)

### Issues:
- fixes #3569 

------
- [x] Updated [changelog](../blob/master/CHANGELOG.md#unreleased)
~- [ ] Updated [migration guide](../blob/master/MIGRATIONS.md#unreleased) if applicable~
~- [ ] Updated [documentation](../blob/master/docs) if applicable~
- [x] Needs datastore update after deployment
- [x] Ready for review

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -241,17 +241,12 @@ services:
...
   # FossilDB
   fossildb:
-    image: scalableminds/fossildb:master__337
+    image: scalableminds/fossildb:master__386
     command:
       - fossildb
       - -c
       - skeletons,skeletonUpdates,volumes,volumeData,volumeUpdates
     user: ${USER_UID:-fossildb}:${USER_GID:-fossildb}
-    healthcheck:
```
Comment (normanrz) [inline docker-compose.yml]: Not required anymore, because it is defined in fossildb's Dockerfile

File: webknossos-datastore/Dockerfile [hunk 2]
```diff
@@ -16,5 +16,11 @@ RUN chown -R webknossos . \
 
 USER webknossos
 
+HEALTHCHECK \
+  --interval=1m --timeout=5s --retries=10 \
+  CMD curl --fail http://localhost:9090/data/health || exit 1
```
Comment (jstriebel) [inline webknossos-datastore/Dockerfile]: Really weird that docker allows the exit-code to be only 0 or 1.

File: webknossos-tracingstore/Dockerfile [hunk 3]
```diff
@@ -17,5 +17,11 @@ RUN chown -R webknossos . \
 
 USER webknossos
 
+HEALTHCHECK \
+  --interval=1m --timeout=5s --retries=10 \
+  CMD curl --fail http://localhost:9090/tracings/health || exit 1
```
Comment (jstriebel) [inline webknossos-tracingstore/Dockerfile]:
```suggestion
  CMD curl --fail http://localhost:9050/tracings/health || exit 1
```

File: webknossos-tracingstore/Dockerfile [hunk 4]
```diff
@@ -17,5 +17,11 @@ RUN chown -R webknossos . \
...
+
+EXPOSE 9090
```
Comment (jstriebel) [inline webknossos-tracingstore/Dockerfile]:
```suggestion
EXPOSE 9050
```

File: webknossos-datastore/Dockerfile (hunk 2 above)
Comment (normanrz) [reply webknossos-datastore/Dockerfile]: Yeah, that's weird.

**Pull request merged**